src/components/Stake/BalanceCard.js

```diff
@@ -28,14 +37,14 @@ function BalanceCard() {
           ${textStyle('title4')};
         `}
       >
-        17,746.21
+        {formatTokenAmount(total, tokenDecimals)}
```

Comment:
Should we be be presenting with trailing zeros on even values?

Reply:
yeah, if they are zeros, formatTokenAmount doesnt display them. But we can submit a PR on aragonUI too change this :)